interface/orders/receive_hl7_results.inc.php

```diff
@@ -857,8 +857,9 @@ function receive_hl7_results(&$hl7, &$matchreq, $lab_id = 0, $direction = 'B', $
             }
 
             $context = $a[0];
+            // StJamesInfirmary: Could come as 'ORU'.
             // Ensoftek: Could come is as 'ORU^R01^ORU_R01'. Handle all cases when 'ORU^R01' is seen.
-            if (strstr($a[8], "ORU^R01")) {
+	    if (strstr($a[8], "ORU") || strstr($a[8], "ORU^R01")) {
```

Comment:
What version HL-7 is distance sending?

Reply:
I would expect to find this at MSH.12, but it looks like the header I am receiving ends at MSH.11

`MSH|^~\&|ApolloLIMS|SF PHL|0194|ST JAMES INFIRMARY|202301041645||ORU|32|P|2.3`

Reply:
Yes it is, there must have been some other issue I was seeing and incorrectly thought this fixed it.